HW02_B_Mimic.md

```diff
@@ -87,3 +117,18 @@ graph below, I used Set2.
 Now using the above mpg dataset, make this graph
 
 ![](HW02_B_Mimic_files/figure-gfm/unnamed-chunk-8-1.png)<!-- -->
+
+```{r graph 4 code}
+ggplot(mpg,aes(class,cty)) +
+  geom_boxplot() +
+  geom_jitter(aes(color = class)) +
+  scale_y_continuous(breaks=seq(0, 35, by = 5)) +
+  labs(title = "Horizontal BoxPlot of City MPG and Car Class", x = "Car Class", y = "City mpg") +
+  theme(panel.border = element_blank(), panel.grid = element_blank(), axis.line = element_line()) +
+  coord_flip() +
+  scale_color_brewer(palette = "Set2")
+
+#I think the black dots are outliers but I have yet to figure out how to exclude them.

```

Comment:
To remove outliers where you call geom_boxplot I added an input of outlier.shape = NA and this seemed to get rid of the black dots [ geom_boxplot(fill = NA, outlier.shape = NA) ] .